plugins/plugin-print/src/load-bitmap-font.ts

```diff
@@ -52,13 +56,70 @@ function parseFont(
   return parseASCII(data);
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function parseNumbersInObject<T extends Record<string, any>>(obj: T) {
+  for (const key in obj) {
+    try {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (obj as any)[key] = parseInt(obj[key], 10);
+    } catch {
+      // do nothing
+    }
+
+    if (typeof obj[key] === "object") {
+      parseNumbersInObject(obj[key]);
+    }
+  }
+
+  return obj;
+}
+
 /**
  *
  * @param bufferOrUrl A URL to a file or a buffer
  * @returns
  */
-async function loadBitmapFontData(bufferOrUrl: string | Buffer) {
-  if (typeof bufferOrUrl === "string") {
+export async function loadBitmapFontData(
+  bufferOrUrl: string | Buffer
+): Promise<LoadedFont> {
+  const isWebWorker = self.document === undefined;
+
+  if (isWebWorker && typeof bufferOrUrl === "string") {
+    const res = await fetch(bufferOrUrl);
+    const text = await res.text();
```

Comment:
The library we use to parse font docs doesn't support workers. This should work in most ends